test/integration/prometheus/ram_bytehours__ram_request_average_costs_analysis_test.go

```diff
@@ -83,7 +90,11 @@ func TestRAMCosts(t *testing.T) {
 
 				// Use this information to find start and end time of pod
 				queryEnd := time.Now().UTC().Truncate(time.Hour).Add(time.Hour)
-				queryStart := queryEnd.Add(-24 * time.Hour)
+				// Get Time Duration
+				timeMumericVal, _ := utils.ExtractNumericPrefix(tc.window)
+				// Assume the minumum unit is an hour
```

Comment:
This is dangerous, if you assume hours, that is fine, but please fail the test if the unit is not hours

Reply:
we are not allowed to pass minutes. Other options are days and special keywords.